rllib/evaluation/worker_set.py

```diff
@@ -281,18 +305,19 @@ def _get_spaces_from_remote_worker(self):
             for e in remote_spaces[0]
         }
 
-        # Try to add the actual env's obs/action spaces.
-        env_spaces = self.foreach_worker(
-            lambda worker: worker.foreach_env(
-                lambda env: (env.observation_space, env.action_space)
-            ),
-            remote_worker_ids=[worker_id],
-            local_worker=False,
-        )
-        if env_spaces:
-            # env_spaces group spaces by environment then worker.
-            # So need to unpack thing twice.
-            spaces["__env__"] = env_spaces[0][0]
+        if not issubclass(self.env_runner_cls, EnvRunner):
```

Comment:
This should be:
`if issubclass(self.env_runner_cls, RolloutWorker):`

Note that also RolloutWorker is a subclass of (the base) `EnvRunner`.